src/components/MultiHopTrade/components/MultiHopTradeConfirm/hooks/useTradeExecution.tsx

```diff
@@ -266,6 +274,9 @@ export const useTradeExecution = (
                 accountNumber,
               } as evm.BuildCustomTxInput)
 
+              if (!receiverAddress) throw Error('Missing receiver address')
+              if (!from) throw Error('Missing sender address')
```

Comment:
`from` wouldn't be caught by runtime checks, since it's not part of `TradeQuote` (now `TradeQuoteOrRate`) but gotten with `adapter.getAddress()` in place, it's more of a paranoia than anything else, we should really not end up here:
https://github.com/shapeshift/web/blob/ddb90afd0214a90361ee646964575ea9dd101f3f/src/components/MultiHopTrade/components/MultiHopTradeConfirm/hooks/useTradeExecution.tsx#L260

However, `receiverAddress` *is* indeed a runtime check that was made possible thanks to types rehaul:

<img width="710" alt="image" src="https://github.com/user-attachments/assets/8523b02e-d5fa-4e99-9f82-df1cb428e2a3">

@woodenfurniture not sure why we would remove the types change, as we wouldn't have gotten tsc screaming at us and forcing us to narrow this in the first place if we did so? Sure, it's *now* guarded at runtime, but only for now in this diff, if we were to remove the types change, we wouldn't have the type safety anymore and would have no safety under our belt in case we touch e.g this file?